Si ejecuto en un directorio donde **NO** est√° la BD en `.beyond/cache/processors.db` el error que me da es:
![Error2](./img/Error2.png)

Cuando reinicio el servicio desde el WS, o sea ya esta la BD, me da:
![Error1](./img/Error1.png)
